tutorials/installation.md

```diff
@@ -25,6 +43,57 @@ wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
 Install Ignition Math:
 
 ```
-sudo apt-get update
-sudo apt-get install libignition-math7-dev
+sudo apt install libignition-math<#>-dev
 ```
+
+Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
+which version you need.
+
+## Source Install
+
+Source installation can be performed in UNIX systems by first installing the
+necessary prerequisites followed by building from source.
+
+### Prerequisites
+
+The optional Eigen component of Ignition Math requires:
+
+  * [Eigen](http://eigen.tuxfamily.org/index.php?title=Main_Page). Refer to the [Eigen Documentation](http://eigen.tuxfamily.org/index.php?title=Main_Page#Documentation) for installation instructions. On Ubuntu systems, `apt-get` can be used to install Eigen:
+
+    ```
+    sudo apt-get install libeigen3-dev
+    ```
+
+The optional Ruby tests of Ignition Math require:
+
+ * [Ruby](https://www.ruby-lang.org/). Refer to the [Ruby Documentation](https://www.ruby-lang.org/downloads/) for installation instructions. On Ubuntu systems `apt-get` can be used to install Ubuntu Package `ruby-dev`:
+
+    ```
+    sudo apt-get install ruby-dev
+    ```
+
+  * [Swig](http://www.swig.org/). Refer to the [Swig Documentation](http://www.swig.org/download.html) for installation instructions. On Ubuntu systems `apt-get` can be used to install Swig:
+
+    ```
+    sudo apt-get install swig
+    ```
+
+### Building from source
+
+1. Clone the repository
+
+    ```
+    git clone https://github.com/ignitionrobotics/ign-math
```

Comment:
Consider adding `-b ign-math<#>` and explaining that the `master` branch has unstable latest code.